### ISSUE TITLE
Selected state of tabs i.e 'Advanced' /'General' in Table/cell/row properties is not clearly visible in High Contrast 'Aquatic' and 'Desert' modes.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8508]
**What is the current behavior? Describe the bug**
Selected state of tabs i.e 'Advanced' /'General' in Table properties in 'Body' edit field should be clearly visible in High Contrast 'Aquatic' and 'Desert' modes. 
Users with vision impairment who rely on High Contrast modes will find it difficult to see which tab is selected if selected state is not shown in High Contrast modes.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**

**Pre-requisites:** Turn on High Contrast (Aquatic/Desert) using Settings->Accessibility->Contrast Themes->Aquatic/Desert.

Step 1: Please open https://codepen.io/keyapanja/pen/JjBMdBg in Edge browser.
Step 2: The default focus would be on HTML body
Step 3: Press ESC, the focus would be on 'Open HTML settings'
Step 4: Navigate to Codepen preview Area.
Step 5: Navigate to Table menu 
Step 6: In Table menu option, invoke Table/cell/row properties.
Step 7: Observe the issue in the Table/cell/row Properties dialog box.

**What is the expected behavior?**
Selected state of tabs i.e 'Advanced/General' should be clearly visible in High Contrast 'Aquatic' and 'Desert' modes.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
OS Version: Windows 11 Dev 22H2(OS Build 22621.1265 )
Edge Dev Browser Version: 110.0.1587.57 (Official build) (64-bit)
Tiny MCE version V6.0

![selected state of tab_normal mode](https://user-images.githubusercontent.com/93735775/222418428-f048895a-a940-48ac-87b4-13094c0e9235.png)
![selected state of tab_HC modes](https://user-images.githubusercontent.com/93735775/222418435-702b36af-2d55-46c5-bdf9-97d4dce2ae30.png)

**Comment by msftedad** [comment]
Hi, Any update on this issue?

**Comment by tiny-ci-servers** [comment]
This issue is stale because it has been open 365 days with no activity. Please comment if you wish to keep this issue open or it will be closed in 30 days.
